workos/mfa.py

```diff
@@ -88,6 +88,30 @@ def enroll_factor(
 
         return WorkOSAuthenticationFactorSms.construct_from_response(response).to_dict()
 
+    def delete_factor(
+        self,
+        id=None,
+    ):
+        """
+        Deletes an MFA authorization factor.
+
+        Kwargs:
+            id (str) - The id of the authorization factor to be deleted.
+
+        Returns: Does not provide a return.
+        """
+
+        if id is None:
+            raise ValueError("Incomplete arguments. Need to specify a factor id.")
+
+        response = self.request_helper.request(
+            "auth/factors/{id}",
+            method=REQUEST_METHOD_POST,
```

Comment:
Should this be a DELETE?

```suggestion
            method=REQUEST_METHOD_DELETE,
```